azure-eventhubs/src/main/java/com/microsoft/azure/servicebus/MessagingFactory.java

```diff
@@ -144,6 +147,37 @@ public CBSChannel getCBSChannel() {
 
         return this.cbsChannel;
     }
+    
+    public ManagementChannel getManagementChannel() {
+    	synchronized (this.mgmtChannelCreateLock) {
+    		if (this.mgmtChannel == null) {
+    			this.mgmtChannel = new ManagementChannel(this, this, "mgmt-link");
+    		}
+    	}
+    	
+    	return this.mgmtChannel;
+    }
+    
+    public void closeManagementChannel() {
```

Comment:
>closeManagementChannel [](start = 16, length = 22)

expose onCloseComplete / offer sync semantics
 - so that the layer above will not have lot of mgmt channels opened (waiting to be closed) at one time.